specs/grafana/SOURCES/grafana-server.service

```diff
@@ -0,0 +1,30 @@
+[Unit]
+Description=Grafana instance
+Documentation=http://docs.grafana.org
+Wants=network-online.target
+After=network-online.target
+After=postgresql.service mariadb.service mysqld.service
```

Comment:
Maybe we should add `clickhouse-server.service` here?

Reply:
Fixed (4ab1291589772c484293d167c0937ffef72612fc)